client/app/scripts/components/node-details.js

```diff
@@ -192,7 +192,11 @@ class NodeDetails extends React.Component {
         <div className="node-details-content">
           {details.metrics && <div className="node-details-content-section">
             <div className="node-details-content-section-header">Status</div>
-            <NodeDetailsHealth metrics={details.metrics} />
+            <NodeDetailsHealth
+              metrics={details.metrics}
+              topologyId={topologyId}
+              nodeColor={nodeColor}
```

Comment:
I went back to the `btn-opacity` which is not as visible but the cursor pointer should be enough once we are hovering.